tests/circuitpython-manual/dotenv_test.env

```diff
@@ -0,0 +1,35 @@
+# No e0 value
+   # comment preceded by spaces
+e1=e1value
+e2=e2value # value followed by a comment
+e3='e3value'
+e4='e4value' # quoted value followed by a comment
+# e5 should be None
+e5
+# e6 should be the empty string
+e6=
+# e7 should be '#' (bash-like syntax processing)
+e7=#
+# e8 should be the empty string
+e8=''
+# e9 should be the empty string
+e9= #
+e10=e10_first
+e10=e10_last
+e11='abc#def'
+# e12 should be 'abc#def'
+e12=abc#def
+e12='multi
+line'
+e13=e13value
+e14  #comment
+e15 = e15value
+# e16 should be '#'
+e16=# #
+# e17 should be 'def#hi'
+e17='def'#hi
+# e18 should be '#has a hash'
+e18=#has a hash
+""")
+
+env_file.close()
```

Comment:
are these lines intended to be here?

Reply:
oops, will force push a fix to that -- bad editing